### PR TITLE
Music + Text Fonts

### PR DESCRIPTION
**Pull request by ronyeh**
Music Fonts are now paired with the recommended Text Fonts.

Tons of visual differences. :-)

I also removed the fallback fonts during testing. For example, I think it is better to know if Leland is missing a glyph. In the past we would just use the Bravura glyph as a fallback.

### CODE REVIEW

File: tests/vexflow_test_helpers.ts [hunk 1]
```diff
@@ -208,17 +208,18 @@ export class VexFlowTests {
...
-    'Finale Maestro': ['Finale Maestro', 'Bravura', 'Academico'],
+    Bravura: ['Bravura', 'Academico'],
+    'Finale Ash': ['Finale Ash', 'Finale Ash Text'],
+    'Finale Broadway': ['Finale Broadway', 'Finale Broadway Text'],
+    'Finale Maestro': ['Finale Maestro', 'Finale Maestro Text'],
+    Gonville: ['GonvilleSmufl', 'Academico'],
+    Gootville: ['Gootville', 'Edwin'],
+    Leland: ['Leland', 'Edwin'],
+    MuseJazz: ['MuseJazz', 'MuseJazz Text'],
+    Petaluma: ['Petaluma', 'Petaluma Script'],
```
Comment (rvilarl) [inline tests/vexflow_test_helpers.ts:222]: Should we add Sebastian and Leipzig?

Comment (rvilarl) [inline tests/vexflow_test_helpers.ts:222]: Should we add Sebastian and Leipzig?

**Review by rvilarl — CHANGES_REQUESTED** [review]

File: tests/vexflow_test_helpers.ts (hunk 1 above)
Comment (ronyeh) [reply tests/vexflow_test_helpers.ts:222]: Sure, I can add them. How long does flow.html tests take on your computer? It is pretty slow on my laptop. At some point we will need to figure out an easy way to run only a subset of the tests / subset of the fonts.

Comment (rvilarl) [reply tests/vexflow_test_helpers.ts:222]: I agree that we should probably have the opportunity to select the fonts when testing.

Comment (rvilarl) [reply tests/vexflow_test_helpers.ts:222]: OK we leave it out for now.

**Pull request merged**